style.css

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 58px 24px;
+  text-decoration: none;
+  color: #6070ff;
+  width: 375px;
+}
+
+.logo {
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+}
+
+main {
+  background-image: url('../Images/header-shapes\ mobile@2x.png');
+  background-repeat: repeat-x;
```

Comment:
- [ ] Good job on styling the image as a background image in CSS instead of in HTML, but the image is not seen as a background image. Kindly fix  : wrench: to be looked at as in the Figma template

Reply:
@Debas-31 Can you shed more light on here, please?
I do not understand what this statement means (but the image is not seen as a background image)